tests/test_simple.py

```diff
@@ -252,55 +252,6 @@ def default_config(self) -> dict[str, Any]:
         return conf
 
 
-class InfoResourceTest(ModuleApiTestCase):
-    async def test_registered_default_info_resource(self) -> None:
-        """Tests that the registered info resource is accessible"""
-
-        channel = self.make_request(
-            method="GET",
-            path="/_synapse/client/com.famedly/tim/v1",
-        )
-
-        assert channel.code == 200, channel.result
-        assert channel.json_body["title"] == "Invite Checker module by Famedly"
-        assert channel.json_body["description"] == "Invite Checker module by Famedly"
-        assert channel.json_body["contact"] == "info@famedly.com"
-        assert channel.json_body["version"], "Version returned"
-
-    @synapsetest.override_config(
-        {
-            "modules": [
-                {
-                    "module": "synapse_invite_checker.InviteChecker",
-                    "config": {
-                        "api_prefix": "/_synapse/client/test",
-                        "title": "abc",
-                        "description": "def",
-                        "contact": "ghi",
-                        "federation_list_url": "https://localhost:8080",
-                        "federation_localization_url": "https://localhost:8000/localization",
-                        "federation_list_client_cert": "tests/certs/client.pem",
-                        "gematik_ca_baseurl": "https://download-ref.tsl.ti-dienste.de/",
```

Comment:
Removing the api_prefix stuff is fine, but imo you should still test the other fields can be modified :)

Reply:
Yeah, not sure why I didn't consider that. I'll add this one back in

Reply:
resolved in [b9dd02b](https://github.com/famedly/synapse-invite-checker/pull/33/commits/b9dd02bde2c0bd6248d0207f6bff29c8c7c4e41b)